app/src/main/java/com/twilio/video/app/ui/room/RoomViewState.kt

```diff
@@ -17,7 +17,7 @@ data class RoomViewState(
     val isCameraEnabled: Boolean = false,
     val localVideoTrack: VideoTrackViewState? = null,
     val isMicEnabled: Boolean = false,
-    val isAudioMuted: Boolean = true,
```

Comment:
Seems like this changes the default. Why was this change needed?

Reply:
So what this does is when the user starts the app and enters the lobby screen, their audio should not be muted by default.

Reply:
Is that how it is for JS and iOS? For all the video apps that we use the default behavior is audio and video are disabled by default. If this change is not required for fixing tests, I would prefer it to be discussed first with @aaalaniz and @donaltoomey.

Reply:
Oh so this is how it has always been in the Android video app if permissions are enabled. The new JS app does the same. This is a fix as it was regressed in the decoupling story a few PRs back.

Reply:
Cool then 👍